client/js/render.js

```diff
@@ -90,6 +99,54 @@ function buildChatMessage(data) {
 	return msg;
 }
 
+function renderPreview(preview, msg) {
```

Comment:
If we do that, then we need to do a `require("client/js/socket-events/msg_preview")` in this file, which I think we shouldn't do. The code in this function is very much related to the renderer, so I think we should leave it there, and modularize the renderer in a later PR if necessary.
For the `click` event, however, I'm less happy to move it here, but we have the same need for `require` otherwise :/